WireAPI/Tests/WireAPITests/__Snapshots__/UsersAPITests/testGetUsersRequest.v0.txt

```diff
@@ -0,0 +1,5 @@
+▿ HTTPRequest
+  ▿ body: Optional<Data>
+    - some: 226 bytes
```

Comment:
Perhaps we need a custom `snapshotDescription` of `HTTPRequest` so we can also verify the body payload.

Reply:
Ah that's possible? :)

Reply:
Done